lte/gateway/python/magma/pipelined/app/egress.py

```diff
@@ -140,10 +140,6 @@ def _get_default_flow_msgs(self, datapath) -> DefaultMsgsMap:
     def initialize_on_connect(self, datapath):
         self._datapath = datapath
         self._setup_non_nat_monitoring()
-        # TODO possibly investigate stateless XWF(no sessiond)
-        if self.config.setup_type == 'XWF':
-            self.delete_all_flows(datapath)
-            self._install_default_flows(datapath)
```

Comment:
is `_install_default_flows` now unused?

Reply:
Yes, your are right. There are no callers within the class anymore.